src/npm/types.ts

```diff
@@ -66,6 +70,12 @@ export interface GetPackage {
   contributors?: Array<{ name: string }>;
   repository?: PackageRepo;
   schematics?: string;
+  types?: string;
+  typings?: string;
+  type?: 'module' | 'commonjs';
+  module?: string;
+  main?: string;
+  exports?: PackageExports;
```

Comment:
I don't think that's true.
The field versions has those values but not the root level.

see for example: https://replicate.npmjs.com/algoliasearch?_rev=413-b9dc0edd639e24292693b6b5d57f8c5c

Reply:
solved in 8578f0f